<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FacesDialog</class>
 <widget class="QDialog" name="FacesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="lyt_color" stretch="0,0">
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_color_label">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorLabel" name="lbl_color_value">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="lyt_scale" stretch="0,1">
       <item>
        <widget class="QLabel" name="lbl_scale_label">
         <property name="text">
          <string>Scale factor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_scale_value">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lyt_neighbours" stretch="0,1">
       <item>
        <widget class="QLabel" name="lbl_neighbours_label">
         <property name="text">
          <string>Min. Neighbours:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_neighbours_value">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>subclasses/color_label.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
